crates/merod/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "merod"
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
@MatejVukosav @miraclx Let's handle the release in a separate PR and include the release notes there. This way, we can incorporate any additional changes and avoid spamming releases unnecessarily.